.github/workflows/release-integration.yml

```diff
@@ -1,10 +1,27 @@
-name: Release
+name: Release metrics adapter image
 
 on:
   release:
     types: [ prereleased, released ]
 
 jobs:
+  # Sometimes chart-releaser might fetch an outdated index.yaml from gh-pages, causing a WAW hazard on the repo
+  # This job checks the remote file is up to date with the local one on release
+#  validate-gh-pages-index:
```

Comment:
cool!, why is commented?

Reply:
It was commented because in the first run the `index.yaml` does not exist so this validation fails and creating it empty makes `cr` fail so it is a cyclic dependency in the first release it is made.

In fact, I have it uncommented right now and it is needed to be commented on during the first execution.